### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 11

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#337] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324886-da24df9e-61be-496f-aeed-b97704857381.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/210001979-cd01e728-641e-4dca-95d6-b2dc14416b6d.png)
